operator/.golangci.yaml

```diff
@@ -1,8 +1,6 @@
 ---
 run:
-  tests: false
-  skip-files:
-    - "example_.+_test.go$"
+  tests: true
```

Comment:
This is the default value, so we could just remove the line from the file. (approving anyway, see this as optional comment)